Model/Transaction/Order.php

```diff
@@ -79,10 +79,6 @@ public function build(
     public function push($forceMethod = null) {
         $orderUpdatedAt = $this->originalOrder->getUpdatedAt();
         $orderSyncedAt = $this->originalOrder->getTjSalestaxSyncDate();
-        $orderApiKey = trim($this->scopeConfig->getValue(TaxjarConfig::TAXJAR_APIKEY,
-            \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
-            $this->originalOrder->getStoreId()
```

Comment:
@ryanreeves-taxjar Don't we still have to pass this store ID to get the correct API token for an order?